core/core-dotspacemacs.el

```diff
@@ -140,6 +140,10 @@ with `:' and Emacs commands are executed with `<leader> :'.")
 (defvaralias 'dotspacemacs-remap-Y-to-y$ 'evil-want-Y-yank-to-eol
   "If non nil `Y' is remapped to `y$'.")
 
+(defvaralias 'dotspacemacs-ex-substitute-global 'evil-ex-substitute-global
+  "If non nil, :substitute changes all matches in a line by default.
+The 'g' flag toggles one or all matches.")
```

Comment:
The first quote of `'g'`should be a backquote to be in line with the rest of the file :-)